packages/flutter/lib/src/material/search_view_theme.dart

```diff
@@ -94,6 +98,8 @@ class SearchViewThemeData with Diagnosticable {
     TextStyle? headerTextStyle,
     TextStyle? headerHintStyle,
     BoxConstraints? constraints,
+    EdgeInsets? padding,
+    bool? shrinkWrap,
```

Comment:
What is shrinkwrap for here?

Reply:
This got missed when I removed all the shrink wrap changes